### PR TITLE
Add a script for checking imports only on .go files

### PR DESCRIPTION
**Pull request by Andreagit97**
# Description

This Script allows to ignore imports of auto-generated files ("zz_generated..")

### CODE REVIEW

File: scripts/utils/goimports_ignore.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+find "$(pwd)" -type f -name '*.go' -a ! -name '*zz_generated*' -exec goimports -local github.com/liqotech/liqo -w {} \;
```
Comment (giorio94) [inline scripts/utils/goimports_ignore.sh:3]: Would move directly to the makefile, as a one-liner

**Review by giorio94 — APPROVED** [review]

**Review by palexster — APPROVED** [review]

File: scripts/utils/goimports_ignore.sh (hunk 1 above)
Comment (Andreagit97) [reply scripts/utils/goimports_ignore.sh]: yes you are right, thank you

**Pull request merged**